README.md

```diff
@@ -2,34 +2,196 @@
+```bash
+php artisan protector:export
 ```
 
+By default the file will be stored in storage/protector and have a timestamp in the name. You can also specify the filename.
+
+You could also automate this by
+- installing a cronjob on linux
+- running it when you deploy to your server
+- creating a Laravel Job and queueing it
+
+```bash
+php artisan protector:export --file=storage/database.sql
+```
+
+### Setup for importing the database of a remote server
+
+This package can run on both servers and client machines of the same software repository. You set up authorized developers on the server, and give them the key for their local machine.
+
+#### Installing protector in your Laravel project
+
+Install the package via composer.
+
+```bash
+composer require cybex/laravel-protector
+```
+
+In your User model class, add the following trait.
+```php
+use Laravel\Sanctum\HasApiTokens;
+
+class User extends Authenticatable
+{
+    use HasApiTokens;
+
+    ...
+}
+```
+
+You can optionally publish the protector config to set options regarding the storage, access and transmission of the files. 
+```bash
+artisan vendor:publish --tag=protector.config
+```
+
+Publish the protector database migration, and optionally modify it to work with your project.
+```bash
+php artisan vendor:publish --tag=protector.migrations
+```
+
+Run the migration on the client and server repository.
+```bash
+php artisan migrate
+```
+
+#### On the Client
+
+Run the following command to receive
+- the public key to give to your server admin
+- the private key to save in your .env file
+```bash
+php artisan protector:keys
+```
+
+Your server admin will then give you the token and server-url to save in your .env file.
+Unless specified otherwise in your software, the .env keys are
+```bash
+PROTECTOR_AUTH_TOKEN=
+PROTECTOR_SERVER_URL=
+```
+
+>Do not give your private key to anyone and keep it protected at all time
+
+See [Usage](#usage) on how to import the remote database.
+
+>Downloaded database dump files are stored unencrypted
+
+#### On the Server
+
+Make sure that the server is accessible to the client machine via HTTPS.
+
+When one of your developers gives you their public key from the previous step, you can authorize them with:
+
+```bash
+php artisan protector:token --publicKey=<public key> <user id>
+```
+
+You will receive the token and url to give back to the developer, who has to save them in their .env file.
+
+The developer can then download and import the server database on their own.
+
+### Setup for collecting backups from multiple servers
+
+You can develop a custom client that can access and store remote server backups. The servers can be different Laravel projects that have the protector package installed.
+
+See the previous chapter on how to give your backup client access to all servers. The backup client will need an according user on each target server.
+* All the backup users on the target servers will have the same public key from the client
```

Comment:
```suggestion
- All the backup users on the target servers will have the same public key from the client
```
It does not make a difference, but you used dashes everywhere else, so it seems odd to use asterisks here.